### PR TITLE
Repeat assignment to flags.normalizeNameFunc

### PR DESCRIPTION
**Pull request by navist2020**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Use `SetNormalizeFunc()` set `f.normalizeNameFunc` continuously.
https://github.com/kubernetes/kubernetes/blob/9ead4bf2ee6be28c0ca830874fa0f14e50d407b5/staging/src/k8s.io/kubectl/pkg/cmd/cmd.go#L476-L480
The first step can be removed.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/cmd.go [hunk 1]
```diff
@@ -473,7 +473,6 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	}
 
 	flags := cmds.PersistentFlags()
-	flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags
```
Comment (knight42) [inline staging/src/k8s.io/kubectl/pkg/cmd/cmd.go:476]: `cliflag.WarnWordSepNormalizeFunc` seems to be different from `cliflag.WordSepNormalizeFunc` at 479 according to their names. Could you help me understand why this line could be removed?

Comment (navist2020) [reply staging/src/k8s.io/kubectl/pkg/cmd/cmd.go:476]: The difference between `WarnWordSepNormalizeFunc` and `WordSepNormalizeFunc` is that the former will output a warning. The function of both is order to set `flags.normalizeNameFunc`. I think the latter(479) code will override the previous(476) result.It's only my personal opinion and i found when i removed it, i can still use `kubectl` normally.

**Pull request closed without merging**